iontestdata/good/equivs/timestampsLargeFractionalPrecision.ion

```diff
@@ -15,4 +15,55 @@
 (
     2007-02-23T12:14:33.18446744073709551616Z // Fraction is max 64 bit unsigned int + 1.
     2007-02-23T12:14:33.18446744073709551616+00:00
-)
\ No newline at end of file
+)
+
+(
+    2007-02-23T12:14:33.99999999999Z // Test rounding up when rounded value greater than 999999.
```

Comment:
How exactly does this test rounding up? Why would the value be rounded up?

Reply:
Using test rounding up is probably a poor choice of words on my part, but essentially this would be truncated to 999999 rather than rounded.

Reply:
It should be neither truncated nor rounded. By the spec, Ion timestamps are capable of supporting arbitrary precision in fractional seconds. Some implementations have restrictions, but precision is not limited by the spec.

The tests themselves are fine, except for the comments; I just want to be clear about the intent of the spec.

Reply:
Done. I removed these comments about rounding/truncating, and re-organized the additional tests a bit. Please review to see if it suits standards.